gopro-rename

```diff
@@ -27,54 +27,55 @@ import ntpath
 import re
 import sys
 
-def GetRenamedFilename(path):
+def GetRenamedFilename(path, prefix):
     dir, file = ntpath.split(path)
 
     match = re.match('GOPR(\d{4})(\..*)', file)
     if match:
-        return os.path.join(dir, 'GoPro_%04d_%02d%s' % (int(match.group(1)), 0, match.group(2)))
+        return os.path.join(dir, prefix + '_%04d_%02d%s' % (int(match.group(1)), 0, match.group(2)))
```

Comment:
`'%s_...' % (prefix, ...)`, so it's safe and consistent

Reply:
Also maybe include `_` into prefix so it's fully customizable.

Reply:
Thanks, will do.

Reply:
Done